test/expr/closure/multi_statement.swift

```diff
@@ -566,3 +566,63 @@ struct Test {
     }
   }
 }
+
+https://github.com/apple/swift/issues/61017
+do {
+  @propertyWrapper
+  struct Wrapper {
+    var wrappedValue: Int
+
+    init(wrappedValue: Int) {
+      self.wrappedValue = wrappedValue
+    }
+  }
+
+  class Test {
+    let bar: () -> Void = {
+      @Wrapper var wrapped = 1
+      let wrapper: Wrapper = _wrapped // Ok
+    }
+  }
+}
+
+https://github.com/apple/swift/issues/61024
+do {
```

Comment:
Took me a while to realize why this link compiles haha (statement label + comment).